packages/image/README.md

```diff
@@ -0,0 +1,108 @@
+# `@sveltejs/image`
+
+**WARNING**: This package is experimental. It uses pre-1.0 versioning and may introduce breaking changes with every minor version release.
+
+This package aims to bring a plug&play image component to SvelteKit that is opinionated enough so you don't have to worry about the details, yet flexible enough for more advanced use cases or tweaks. It uses the `srcset` and `sizes` attributes of the `img` tag to provide resized images suitable for various device sizes, which for example results in smaller images downloaded for mobile.
+
+## Setup
+
+Install:
+
+```bash
+npm install --save @sveltejs/image
+```
+
+Adjust `vite.config.js`:
+
+```diff
++import { vitePluginSvelteImage } from '@sveltejs/image/vite';
+import { sveltekit } from '@sveltejs/kit/vite';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+	plugins: [
++		vitePluginSvelteImage({ providers: { default: '@sveltejs/image/providers/<choose one>' } }),
+		sveltekit()
+	]
+});
+```
+
+> `<choose one>` refers to chosing one of the ready-to-use providers. We plan to add more providers over time. You can create your own by creating a JavaScript with a `export function getURL({ src, width, height }): string` function inside.
+
+In case of Vercel, adjust `svelte.config.js`:
+
+```diff
+import adapter from '@sveltejs/adapter-vercel';
+
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+	kit: {
+-		adapter: adapter()
++		adapter: adapter({ images: true })
+	}
+};
+
+export default config;
+```
+
+## Usage
+
+### When using one of the providers, i.e. an image CDN:
+
+```svelte
+<script>
+	import Image from '@sveltejs/image';
+</script>
+
+<Image src="/path/to/your/image.jpg" width={1200} height={1800} alt="An alt text" />
+```
+
+`width` and `height` should be the natural width/height of the referenced image. `alt` should describe the image. All are required. The `src` is transformed by calling `getURL` of the `default` provider provided in the `vite.config.js`.
+
+### Static build time optimization:
+
+```svelte
+<script>
+	import Image from '@sveltejs/image';
+    import YourImage from '/path/to/your/image.jpg?generate';
```

Comment:
I'd expect this to be provided by `defaultDirectives` as having to put it manually feels a bit clunky. Users should still be able to override it in the case. I'd also separate it into two pieces and use the standard directives. I'd make it `?w=100,200,400&as=img`. Then we can later have `?w=100,200,400&as=picture` (`as=img` will be built-in for the next major release). Doing it this way should also make it easy for users to override

Also, this line should be updated from spaces to tabs